e2e/e2e_canary_upgrade_test.go

```diff
@@ -0,0 +1,299 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/stackrox/acs-fleet-manager/fleetshard/pkg/central/operator"
+	"github.com/stackrox/acs-fleet-manager/internal/dinosaur/constants"
+	"github.com/stackrox/acs-fleet-manager/internal/dinosaur/pkg/api/public"
+	"github.com/stackrox/acs-fleet-manager/internal/dinosaur/pkg/services"
+	"github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager"
+	"github.com/stackrox/acs-fleet-manager/pkg/features"
+	"github.com/stackrox/rox/operator/apis/platform/v1alpha1"
+	"gopkg.in/yaml.v2"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	namespace = "acsms"
+)
+
+var _ = Describe("Fleetshard-sync Targeted Upgrade", func() {
+
+	BeforeEach(func() {
+
+		if !runCanaryUpgradeTests || !features.StandaloneMode.Enabled() {
```

Comment:
The test can run always when the targeted upgrades feature is enabled, at least in the e2e test suite. 

```suggestion
		if feature.TargetedUpgrades.Enabled() || !features.StandaloneMode.Enabled() {
```